app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,221 @@
+
+            <% if last_name_was_invalid %>
+              <% resource.errors.full_messages_for(:last_name).each do |message| %>
+                <div class="invalid-feedback d-flex">
+                  <%= message %>
+                </div>
+              <% end %>
+            <% end %>
+          </div>
+          <!-- Devise Input for last_name end -->
+          
+          <!-- Devise Input for profile_picture start -->
+          <div class="form-group">
+            <%= f.label :profile_picture, style: "display:block" %>
+            <%= f.file_field :profile_picture, class:"form-control" %>
+          </div>
+
+          <!-- Devise Input for profile_picture end -->
+          
+          <!-- Devise Input for bio start -->
+          <div class="form-group">
+            <% bio_was_invalid = resource.errors.include?(:bio) %>
+
+            <% bio_class = "form-control" %>
+
+            <% if was_validated %>
+              <% if bio_was_invalid %>
+                <% bio_class << " is-invalid" %>
+              <% else %>
+                <% bio_class << " is-valid" %>
+              <% end %>
+            <% end %>
+
+            <%= f.label :bio %>
+
+            <%= f.text_area :bio, :class => bio_class, :placeholder => "Enter bio" %>
+
+            <% if bio_was_invalid %>
+              <% resource.errors.full_messages_for(:bio).each do |message| %>
+                <div class="invalid-feedback d-flex">
+                  <%= message %>
+                </div>
+              <% end %>
+            <% end %>
+          </div>
+          <!-- Devise Input for bio end -->
+
+          <!-- Devise Input for preferred_timezone start -->
+          <div class="form-group">
+            <% preferred_timezone_was_invalid = resource.errors.include?(:preferred_timezone) %>
+
+            <% preferred_timezone_class = "form-select" %>
+
+            <% if was_validated %>
+              <% if preferred_timezone_was_invalid %>
+                <% preferred_timezone_class << " is-invalid" %>
+              <% else %>
+                <% preferred_timezone_class << " is-valid" %>
+              <% end %>
+            <% end %>
+
+            <%= f.label :preferred_timezone %>
+
+            <%= f.select :preferred_timezone, User.preferred_timezones.keys.map{ |key| [key.upcase, key] }, { selected: @user.preferred_timezone|| :draft }, { :class => preferred_timezone_class, :placeholder => "Enter your preferred timezone" } %>
+
+            <% if preferred_timezone_was_invalid %>
+              <% resource.errors.full_messages_for(:preferred_timezone).each do |message| %>
+                <div class="invalid-feedback d-flex">
+                  <%= message %>
+                </div>
+              <% end %>
+            <% end %>
+          </div>
+          <!-- Devise Input for preferred_timezone end -->
+
+          <%= f.button class: "btn btn-outline-primary btn-block" %>
```

Comment:
I think you should edit your button text from 'Create User' to 'Sign Up' for better ux.

Reply:
See commits: 
c866a63- UI changes based off Amanda's feedback
c2bdaf9- Layout change for create pair button